ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -392,20 +392,22 @@ private void OnResponseReceived(int requestId, HttpResponseMessage response)
         {
             var headerData = new JObject();
 #if WINDOWS_UWP
-            TranslateHeaders(headerData, response.Headers);
+            var responseHeaders = response.Headers;
 #else
-            IDictionary<string, string> headers = response.Headers.ToDictionary((kvp) => kvp.Key, (kvp) => kvp.Value.ToString());
-            TranslateHeaders(headerData, headers);
+            var responseHeaders = response.Headers.Select(pair => 
+                new KeyValuePair<string, string>(pair.Key, string.Join(", ", pair.Value)));
```

Comment:
@kevinvangelder I presume this is the expected behavior for the list of headers?  In System.Net.Http, the header values are an IEnumerable<string> rather than a single string, so I figured header values should just be concatenated with a comma.